azure-pipelines.yml

```diff
@@ -0,0 +1,22 @@
+resources:
+- repo: self
+
+pool:
+  vmimage: 'ubuntu-16.04'
```

Comment:
NIT: You could change this to `vmImage: 'Ubuntu-16.04'` instead

Reply:
If you have a strong preference I'll change it, but we're preferring the `ubuntu-16.04` format in [our](https://docs.microsoft.com/en-us/azure/devops/pipelines/process/variables?view=azure-devops&tabs=yaml%2Cbatch) [various](https://docs.microsoft.com/en-us/azure/devops/pipelines/get-started-multiplatform?view=azure-devops) [docs](https://docs.microsoft.com/en-us/azure/devops/pipelines/process/container-phases?view=azure-devops&tabs=yaml) and [blog posts](https://docs.microsoft.com/en-us/azure/devops/pipelines/get-started-multiplatform?view=azure-devops) and samples.

Reply:
Oh! I don't have a strong preference as such, we followed this convention in `azure-pipelines-yaml` repository. But do change `vmimage` to `vmImage` though.

Reply:
Updated.